internal/cli/capact/components.go

```diff
@@ -648,12 +645,23 @@ func createObject(configuration *action.Configuration, content []byte) error {
 		return errors.Wrap(err, "while validating the object")
 	}
 
-	if _, err := configuration.KubeClient.Create(res); err != nil {
-		// If the error is CRD already exists, return.
-		if apierrors.IsAlreadyExists(err) {
-			return nil
-		}
-		return errors.Wrapf(err, "while creating the object")
+	// 5 exponential retries: ~2µs ~12ms ~70ms ~325ms ~1.63s
+	retryBackoff := retry.DefaultBackoff
+	retryBackoff.Steps = 5
+	err = retry.OnError(retryBackoff, differentThanAlreadyExistErr, func() error {
+		_, err = configuration.KubeClient.Create(res)
+		// You have to return err itself here (not wrapped inside another error)
+		// so we can identify it correctly.
+		return err
+	})
+	if err != nil && !apierrors.IsAlreadyExists(err) {
```

Comment:
If we have such function already... 😄 

```suggestion
	if err != nil && differentThanAlreadyExistErr(err error) {
```